package/MDAnalysis/guesser/base.py

```diff
@@ -113,6 +109,13 @@ def is_guessable(self, guess):
                     return False
             return True
         return False
+    
+    def is_value_missing(self, value, attr_class):
+        """check if an attribute value is equal to missing_value_label value"""
+        if hasattr(attr_class, 'missing_value_label'):
+            return value == attr_class.missing_value_label or np.isnan(value)
+        else:
+            return False
```

Comment:
I was thinking about having this method in the base class for topology attributes. This way the `missing_value_label` and its use are kept in one place. Also, it may be useful beyond guessers: we can later on think about using `select_atoms` to get the atoms with missing mass or missing charge...